packages/shared/auth0-client/src/index.ts

```diff
@@ -181,6 +193,26 @@ export default class Auth0Client {
     });
   }
 
+  private getInstanceWithCredentials(username: string, password: string): Promise<AxiosInstance> {
```

Comment:
I've called it `email` elsewhere (i.e. the API client) and `username` here because we're in generic auth0-client land at this point, which doesn't/shouldn't know about business decisions that sit outside of it. 

The field you send to auth0 is `username` even if only emails are enabled.

Reply:
Fair enough.